packages/fractal/fractal/src/fractal.test.js

```diff
@@ -123,15 +121,16 @@ describe('Fractal', function () {
       const fractal = makeFractal();
```

Comment:
I can't leave a comment on the exact line as there were no changes, but this applies to `'Can render components'` - as discussed, we should just test the component-rendering aspect, so the collection should be mocked via `getParser`